src/Roslyn.Diagnostics.Analyzers/Core/DoNotCopyValue.cs

```diff
@@ -9,6 +9,8 @@
 using Analyzer.Utilities;
 using Analyzer.Utilities.Extensions;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Not recommended in the shared layer. Please move to the C# layer.

Reply:
Is there a language independent way to get the `GetEnumerator` method used by the `IForEachLoopOperation`?

Reply:
➡️ Moved language-specific functionality to derived types